network-api/networkapi/wagtailpages/models.py

```diff
@@ -586,25 +587,32 @@ def filter_entries_for_tag(self, entries, context):
         specific model before we can test for whether i) there are tags
         to work with and then ii) those tags match the specified ones.
         """
-        terms = self.filtered.get('terms')
-
-        # "unsluggify" all terms:
-        context['terms'] = [str(Tag.objects.get(slug=term)) for term in terms]
-
-        entries = [
-            entry
-            for
-            entry in entries.specific()
-            if
-            hasattr(entry, 'tags')
-            and not
-            # Determine whether there is any overlap between 'all tags' and
-            # the tags specified. This effects ANY matching (rather than ALL).
-            set([tag.slug for tag in entry.tags.all()]).isdisjoint(terms)
-        ]
+        type = self.filtered.get('type')
```

Comment:
We probably need to revisit this change, as it's on the merged master now. It might not be correct anymore.

Reply:
I updated this line to `terms = self.filtered.get('terms')` and removed lines 591-594 to match your category blog pr changes. Is that correct or should L590 be the only thing updated here?